src/main/resources/application.properties

```diff
@@ -1,2 +1,5 @@
 spring.main.web-environment=false
-spring.data.mongodb.database=submissions
\ No newline at end of file
+spring.data.mongodb.database=submissions
+usi.tokenservice.username=${usiCoreAAPUsername}
+usi.tokenservice.password=${usiCoreAAPPassword}
+usi.apiRootUrl=http://localhost:8080/api
```

Comment:
the api root is also defined in subs-api like this: server.contextPath=/api
Do we need both of them?

Reply:
They serve different purposes. `server.contextPath` is for the server configuration, `usi.apiRootUrl` is config for the client